server/models/wave.py

```diff
@@ -1,18 +1,20 @@
-from sqlalchemy import Column, Float, Integer, String
+from sqlalchemy import Column, DateTime, Float, ForeignKey, Integer, String
 
 from server.db.base_class import Base
 
 
-class Historical_Wave(Base):
-    __tablename__ = "Historical Wave Data"
+class Wave(Base):
+    __tablename__ = "Wave"
```

Comment:
Set this lowercase for consistency with all our other tables.

`__tablename__ = "wave"`